sites/partners/cypress/integration/07-jurisdictional-admin-user-management.spec.ts

```diff
@@ -1,6 +1,6 @@
 describe("Admin User Mangement Tests", () => {
   before(() => {
-    cy.login("jurisdictionalAdmin")
+    cy.loginAndAcceptTerms("jurisdictionalAdmin")
```

Comment:
Was this a change required by this PR or was this broken on dev?

Reply:
I think this change came about because of this PR since other PRs were able to pass. I think its because of the change to no longer having many migrations and instead only having 1 that user now has to accept terms whereas before they didn't have to